docs/source/workflow/generate-types.mdx

```diff
@@ -13,56 +13,47 @@ You can use these generated TS types in your resolvers to type-check that your r
 
 ## Setting up your project
 
-We'll use the [GraphQL Code Generator](https://www.the-guild.dev/graphql/codegen) library to generate types based on our GraphQL schema. There are [multiple ways](https://www.the-guild.dev/graphql/codegen/docs/config-reference/schema-field) to provide a schema to GraphQL Code Generator. Below, we'll show the most common method, which requires our schema to be in a `.graphql` file.
+We'll use the [GraphQL Code Generator](https://www.the-guild.dev/graphql/codegen) library to generate types based on our GraphQL schema. There are [multiple ways](https://www.the-guild.dev/graphql/codegen/plugins/typescript/typescript-resolvers#usage-example) to provide a schema to GraphQL Code Generator. Below, we'll show one of the common methods, beginning by wrapping our schema in a `gql` template literal tag from the [`graphql-tag`](https://www.npmjs.com/package/graphql-tag) package.
 
-If you haven't already, move your server's schema into a `.graphql` file, like so:
+If you haven't already, begin by importing the `graphql-tag` library:
 
-```graphql title="src/schema.graphql"
-type Query {
-  books: [Book]
-}
+```bash
+npm install graphql-tag
+```
 
-type Book {
-  title: String
-  author: String
-}
+The `gql` template literal tag from `graphql-tag` can parse a Schema Definition Language (SDL) string into an abstract syntax tree (AST). Under the hood, Apollo Server handles this for you when you provide your schema as a string. 
 
-type AddBookMutationResponse {
-  code: String!
-  success: Boolean!
-  message: String!
-  book: Book
-}
+If we pass GraphQL Code Generator a TypeScript file path, it will automatically generate types for the `gql` tag within that file. So, we can wrap our schema string in a `gql` tag to let GraphQL Code Generator know it should generate types for that tag. 
 
-type Mutation {
-  addBook(title: String, author: String): AddBookMutationResponse
-}
-```
+Import the `gql` tag into the file where you define your schema, then wrap your schema string like so:
 
-If you moved your schema into a `.graphql` file, update your imports to ensure you're still properly passing your schema to your server. In the file where you create your server, you can read in your schema using `readFileSync` from the `fs` package:
+```ts title="src/schema.ts"
+import gql from 'graphql-tag';
 
-```ts title="src/index.ts"
-// ...other imports
-import { readFileSync } from 'fs';
-import path, { dirname } from 'path';
-import { fileURLToPath } from 'url';
-
-// If you are using ESM module syntax you can redefine `__dirname` as 
-// shown below. If you are using CJS syntax you can use `__dirname` directly.
-const __dirname = dirname(fileURLToPath(import.meta.url));
-const schemaFile = path.join(__dirname, '../src/schema.graphql');
-// Once we have our schema's file path, read in our schema. 
-const typeDefs = readFileSync(schemaFile, { encoding: 'utf-8' });
+export default gql(`
```

Comment:
No surrounding parens needed for the gql tag, like so:

```ts
export default gql`
  ...
`;
```

Reply:
(Fixing this here and in the corresponding `docs-example`